### ISSUE TITLE
Loss Fluctuation in YOLOv4

### ISSUE THREAD

**Issue by mfaramarzi** [AlexeyAB/darknet#5527]
@AlexeyAB 

Hi Alex,

Can you tell me why Loss values have considerably more fluctuation than YOLOv3?

YOLOv3:

![image](https://user-images.githubusercontent.com/44882423/81358433-7af2f300-90a4-11ea-81f7-d6ea334571f4.png)

YOLOv4:

![image](https://user-images.githubusercontent.com/44882423/81358451-834b2e00-90a4-11ea-9579-1c1eab49cd09.png)

**Comment by beizhengren** [comment]
Hello @mfaramarzi, would you share me your yolov4-coustom.cfg? my loss function curve not converge.
Thank you.

**Comment by mfaramarzi** [comment]
@beizhengren 
Create file yolo-obj.cfg with the same content as in yolov4-custom.cfg (or copy yolov4-custom.cfg to yolo-obj.cfg)
did you do it?

**Comment by beizhengren** [comment]
@mfaramarzi 
Yes I copied yolov4-custom.cfg and modified classes and filters as my own cfg file.
I see your loss curve going down rapidly at about 800 iterations, while my own keeps flatten at almost 8000 iterations.
BTW, when I train based on early saved weights, such as 8000.weights, the char.png as following will not show the curve less than 8000 iterations. How to show that?
Thank you.

**Comment by mfaramarzi** [comment]
> @AlexeyAB
> 
> Hi Alex,
> 
> Can you tell me why Loss values have considerably more fluctuation at YOLOv4 than YOLOv3?
> ### **Tiny-YOLOv3:**
> 
> ![image](https://user-images.githubusercontent.com/44882423/81358433-7af2f300-90a4-11ea-81f7-d6ea334571f4.png)
> ### **YOLOv3:**
> 
> ![image](https://user-images.githubusercontent.com/44882423/81358575-f18ff080-90a4-11ea-854e-3aa1b5fdf026.png)
> ### **YOLOv4:**
> 
> ![image](https://user-images.githubusercontent.com/44882423/81358451-834b2e00-90a4-11ea-9579-1c1eab49cd09.png)

@AlexeyAB 
Alex. I would really appreciate it if you kindly help me with that

**Comment by AlexeyAB** [comment]
> Can you tell me why Loss values have considerably more fluctuation at YOLOv4 than YOLOv3?

Because YOLOv4 uses more agressive training - several anchors per one truth and mosaic data augmentation.
Loss doesn't matter.
